### ISSUE TITLE
[QA] The accuracy of training gcn model on arxiv dataset is too low

### ISSUE THREAD

**Issue by Sanzo00** [alibaba/GraphScope#1904]
**Describe the bug**
I'm new to graphscope, I try to train a gcn model on arxiv dataset, but the accuracy I get is low, I don't know if it's a problem with parameter settings?

![image](https://user-images.githubusercontent.com/36565277/181863205-cae2f9c5-7432-4e6d-9e68-78d7a18bfa47.png)

**To Reproduce**
```py
# Import the graphscope module.
import graphscope
import time

graphscope.set_option(show_log=False)  # enable logging
# graphscope.set_option(log_level='ERROR')

# Load arxiv dataset
from graphscope.dataset import load_ogbn_arxiv
graph = load_ogbn_arxiv()
print(graph)

paper_features = []
for i in range(128):
    paper_features.append("feat_" + str(i))

# launch a learning engine.
lg = graphscope.graphlearn(
    graph,
    nodes=[("paper", paper_features)],
    edges=[("paper", "citation", "paper")],
    gen_labels=[
        ("train", "paper", 100, (0, 75)),
        ("val", "paper", 100, (75, 85)),
        ("test", "paper", 100, (85, 100)),
    ],
)

import graphscope.learning
from graphscope.learning.examples import GCN
from graphscope.learning.graphlearn.python.model.tf.optimizer import get_tf_optimizer
from graphscope.learning.graphlearn.python.model.tf.trainer import LocalTFTrainer

# supervised GCN.
def train(config, graph):
    def model_fn():
        return GCN(
            graph,
            config["class_num"],
            config["features_num"],
            config["batch_size"],
            val_batch_size=config["val_batch_size"],
            test_batch_size=config["test_batch_size"],
            categorical_attrs_desc=config["categorical_attrs_desc"],
            hidden_dim=config["hidden_dim"],
            in_drop_rate=config["in_drop_rate"],
            neighs_num=config["neighs_num"],
            hops_num=config["hops_num"],
            node_type=config["node_type"],
            edge_type=config["edge_type"],
            full_graph_mode=config["full_graph_mode"],
        )

    graphscope.learning.reset_default_tf_graph()
    trainer = LocalTFTrainer(
        model_fn,
        epoch=config["epoch"],
        optimizer=get_tf_optimizer(
            config["learning_algo"], config["learning_rate"], config["weight_decay"]
        ),
    )

    trainer.train_and_evaluate()

# define hyperparameters
config = {
    "class_num": 40,  # output dimension
    "features_num": 128,
    # "features_num": 1433,
    "batch_size": 200,
    "val_batch_size": 500,
    "test_batch_size": 1000,
    "categorical_attrs_desc": "",
    "hidden_dim": 128,
    "in_drop_rate": 0.5,
    "hops_num": 2,
    "neighs_num": None,
    # "neighs_num": [25, 10],
    "full_graph_mode": True,
    # "full_graph_mode": False,
    "agg_type": "gcn",  # mean, sum
    "learning_algo": "adam",
    "learning_rate": 0.01,
    "weight_decay": 0.0005,
    "epoch": 200,
    "node_type": "paper",
    "edge_type": "citation",
}

train(config, lg)

```
